ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -393,11 +393,22 @@ export default defineComponent({
       })
     })
 
+    const cleanUpInput = (dirtyValue: string|undefined) => {
+      if (dirtyValue === undefined) {
```

Comment:
Could this be simplified? ie `(!dirtyValue)`
Checking for undefined is pretty explicit. What is the argument was `null`?

Reply:
Ok, fair enough! Thanks for the explanation 👍